src/routes/home/HomeScreen.tsx

```diff
@@ -56,6 +60,21 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
     })
   }
 
+  const navigateToNextExercise = async (item: Discipline): Promise<void> => {
+    getNextExercise(item)
+      .then(async nextExerciseData => {
+        const [disciplineId, exerciseKey] = nextExerciseData
```

Comment:
Better to use an object as return type, otherwise it is just `Promise<number, number>` and one does not know which numbers it returns.